corehq/apps/users/bulkupload.py

```diff
@@ -277,7 +277,7 @@ def create_or_update_groups(domain, group_specs, log):
     group_names = set()
     for row in group_specs:
         group_id = row.get('id')
-        group_name = row.get('name')
+        group_name = unicode(row.get('name') or '')
```

Comment:
For anyone reading this later, looks like [openpyxl automatically converts strings to unicode](https://groups.google.com/d/msg/openpyxl-users/yKCaXyoYSVo/2kz83Dww8g8J), so no need to worry about encodings here.